### DISCUSSION TITLE
Overclock on rg350 dont work. Crazy screen on all emulators.

### DISCUSSION THREAD

**Discussion by abrusco** [eduardofilo/RG350_adam_image#83] [Hardware problems?]
![20210822_123158_003](https://user-images.githubusercontent.com/8458577/130361028-80236f81-5b07-45a2-9814-5484deb30f1d.jpg)

**Comment by eduardofilo** [comment]
This is not an image issue. It is the called "silicon lotto". Some processors support the OC and some others don't.

**Comment by abrusco** [comment]
Maybe change OC values in  simple menu config.ini?

**Comment by eduardofilo** [comment]
I think that the values in image are reasonable:

```
;Values (in MHz)
; - 444
; - 492
; - 600
; - 696
; - 792
; - 900
; - 996
; - 1092
; - 1200
[CPU]
underclocked_speed = 444
normal_speed = 996
overclocked_speed = 1092
sleep_speed = 444
```

If overclocked_speed were 1200, I would agree to change, but 1092 is the minimal overclock over the normal speed (996). It is a lotto.

**Comment by abrusco** [comment]
> I think that the values in image are reasonable:
> 
> ```
> ;Values (in MHz)
> ; - 444
> ; - 492
> ; - 600
> ; - 696
> ; - 792
> ; - 900
> ; - 996
> ; - 1092
> ; - 1200
> [CPU]
> underclocked_speed = 444
> normal_speed = 996
> overclocked_speed = 1092
> sleep_speed = 444
> ```
> 
> If overclocked_speed were 1200, I would agree to change, but 1092 is the minimal overclock over the normal speed (996). It is a lotto.

Yes, is good value. I try 1024 And I had the same result. It's a lotto anyway

**Comment by eduardofilo** [comment]
1024 isn't supported. You can only use the values from the list.
